projects/label-designer/src/lib/label-designer.interface.ts

```diff
@@ -281,8 +281,13 @@ export interface IColumnMap {
   [col: string]: string;
 }
 
+/**
+ * @internal
+ */
+export type LabelDataValue = string|number|boolean|string[];
+
```

Comment:
I removed that because I don't think it needs to be internal. That comment is used in other places of label designer, here is documentation related to it: https://www.typescriptlang.org/tsconfig#stripInternal.